contracts/HybridCustody.cdc

```diff
@@ -271,6 +271,18 @@ pub contract HybridCustody {
         // capabilities.
         pub var filter: Capability<&{CapabilityFilter.Filter}>?
 
+        // display is its own field on the ChildAccount resource because only the parent should be able to set this
+        // field.
+        pub let childAccountDisplays: {Address: MetadataViews.Display}
+
+        pub fun setChildAccountDisplay(address: Address, _ d: MetadataViews.Display) {
```

Comment:
We either need a way for a display to be removed on this method (let it be an optional display and remove from the dictionary if nil, or we should add a new method